docs/advanced.md

```diff
@@ -449,6 +449,20 @@ yargs.parserConfiguration({
 See the [yargs-parser](https://github.com/yargs/yargs-parser#configuration) module
 for detailed documentation of this feature.
 
+## Command finish hook
+### Example
+```
+yargs
+    .comand('cmd', ..., async () => {
```

Comment:
Typo, should be `command` not `comand`

Reply:
Thanks @dapplion. Mind doing a small PR to fix this typo?

Reply:
> Thanks @dapplion. Mind doing a small PR to fix this typo?

Sure @mleguen: https://github.com/yargs/yargs/pull/1506